File: anul 2/C++/lab1_1.cpp
#include<stdlib.h>
#include<iostream>

using namespace std;

class Student {

    public:
    string nume,prenume,grupa,pastrat,imbunatatiri,dezvoltarePersonala,dupaFacultate;
};

int main()
{
    Student student;
    student.nume ="Paliciuc";
    student.prenume ="Cosmin-Constantin";
    student.grupa ="323AC";
    student.pastrat ="Formatul laboratorului";
    student.imbunatatiri ="nimic";
    student.dezvoltarePersonala ="Cursuri de 3D modelling";
    student.dupaFacultate="Full stack";
    cout<<student.nume<<endl<<student.grupa<<endl<<student.pastrat<<endl<<student.imbunatatiri<<endl<<student.dezvoltarePersonala<<endl<<student.dupaFacultate;
}

File: anul 2/poo/lab1/Lab_1_Paliciuc_Cosmin-Constantin_323AC/kotlin/lab1_1.kt
class Student {
    var nume: String? = null
    var prenume: String? = null
    var grupa: String? = null
    var pastrat: String? = null
    var imbunatatiri: String? = null
    var dezvoltarePersonala: String? = null
    var dupaFacultate: String? = null
}


fun main(args: Array<String>) {
    var student=Student()
    student.nume ="Paliciuc"
    student.prenume ="Cosmin-Constantin"
    student.grupa ="323AC"
    student.pastrat ="Formatul laboratorului"
    student.imbunatatiri ="nimic"
    student.dezvoltarePersonala ="Cursuri de 3D modelling"
    student.dupaFacultate="Full stack"
    println(student.nume)
    println(student.grupa)
    println(student.pastrat)
    println(student.imbunatatiri)
    println(student.dezvoltarePersonala)
    println(student.dupaFacultate)
}